app/backup.go

```diff
@@ -53,6 +54,46 @@ func BackupCreateCmd() cli.Command {
 	}
 }
 
+func BackupStatusCmd() cli.Command {
+	return cli.Command{
+		Name:  "status",
+		Usage: "query the progress of the backup: status <backupID>",
+		Action: func(c *cli.Context) {
+			if err := checkBackupProgress(c); err != nil {
+				logrus.Fatalf("Error querying backup status: %v", err)
+			}
+		},
+	}
+}
+
+func checkBackupProgress(c *cli.Context) error {
+	backupID := c.Args().First()
+	if backupID == "" {
+		return fmt.Errorf("Missing required parameter backupID")
+	}
+
+	client := getCli(c)
+	replicaIP, err := client.GetReplicaIP(backupID)
+	if err != nil {
+		logrus.Errorf("Error fetching the ReplicaIP")
+		return err
+	}
+	if replicaIP == "" {
+		return fmt.Errorf("replica IP is empty")
+	}
+
+	//Fetch backupObject using the replicaIP
+	task := sync.NewTask(c.GlobalString("url"))
+
+	backupStatus, err := task.FetchBackupStatus(backupID, replicaIP)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(string(backupStatus))
```

Comment:
remove it or use `logrus`

Reply:
This is required as the output of the command executed by engine will be read by manager periodically every 5 seconds.